_layouts/topic.html

```diff
@@ -50,7 +50,7 @@ <h2 id="material">Material</h2>
             {% endunless %}
             {% if topic.subtopics %}
               {% for subtopic in topic.subtopics %}
-                <h3>{{ subtopic.title}}</h3>
+                <h3 id="st-{{ subtopic.title }}">{{ subtopic.title}}</h3>
```

Comment:
do we need to `| slugify` this in case there are spaces in title?

Reply:
ahhh you're brilliant, yep, we do. Good call.